articles/defender-for-cloud/iac-vulnerabilities.md

```diff
@@ -188,256 +188,12 @@ Once you have set up the Microsoft Security DevOps GitHub action or Azure DevOps
-
-Service Fabric clusters should only use Azure Active Directory for client authentication. A Service Fabric cluster offers several entry points to its management functionality, including the web-based Service Fabric Explorer, Visual Studio and PowerShell. Access to the cluster must be controlled using AAD.
-
-**Recommendation**: [Enable AAD client authentication on your Service Fabric clusters](../service-fabric/service-fabric-cluster-creation-setup-aad.md)
-
-**Severity level**: 1
-
-#### TA-000027: Transparent Data Encryption on SQL databases should be enabled
-
-Transparent data encryption should be enabled to protect data-at-rest and meet compliance requirements.
-
-**Recommendation**: To [enable transparent data encryption](/azure/azure-sql/database/transparent-data-encryption-tde-overview?tabs=azure-portal), in the [Microsoft.Sql/servers/databases/transparentDataEncryption resource properties](/azure/templates/microsoft.sql/servers/databases/transparentdataencryption?tabs=json), add (or update) the value of the *state* property to `enabled`.
-
-**Severity level**: 3
-
-#### TA-000028: SQL servers with auditing to storage account destination should be configured with 90 days retention or higher
-
-Set the data retention for your SQL Server's auditing to storage account destination to at least 90 days.
-
-**Recommendation**: For incident investigation purposes, we recommend setting the data retention for your SQL Server's auditing to storage account destination to at least 90 days, in the [Microsoft.Sql/servers/auditingSettings resource properties](/azure/templates/microsoft.sql/2020-11-01-preview/servers/auditingsettings?tabs=json#serverblobauditingpolicyproperties-object), using the *retentionDays* property. Confirm that you're meeting the necessary retention rules for the regions in which you're operating. This is sometimes required for compliance with regulatory standards.
-
-**Severity level**: 3
-
-#### TA-000029: Azure API Management APIs should use encrypted protocols only
-
-Set the protocols property to only include HTTPS.
-
-**Recommendation**: To use encrypted protocols only, add (or update) the *protocols* property in the [Microsoft.ApiManagement/service/apis resource properties](/azure/templates/microsoft.apimanagement/service/apis?tabs=json), to only include HTTPS. Allowing any other protocols (for example, HTTP, WS) is insecure.
-
-**Severity level**: 1
+IaC files are analyzed in Microsoft Security DevOps through multiple tools. Information about the rules for ARM and Bicep templates included as part of TemplateAnalyzer, along with remediation details, can be found [here](https://github.com/Azure/template-analyzer/blob/development/docs/built-in-bpa-rules.md#built-in-rules). While details about the multi-cloud rules covered by Terrascan can be found [here](https://runterrascan.io/docs/_print/#pg-07a53d0ea8f57b62a06a9235598d5c55).
```

Comment:
For Terrascan, thoughts on using this link instead: `https://runterrascan.io/docs/policies/` ?